app/models/repo.rb

```diff
@@ -108,6 +110,11 @@ def weight
     end
   end
 
+  def amount_code_helpers
+    apiResponse = Net::HTTP.get_response(URI.parse("https://api.github.com/repos/#{full_name}/contributors"))
```

Comment:
I think the model should not be responsible to implement an external request. Maybe you can use a service object or an interactor to implement it. What do you think?